hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java

```diff
@@ -355,8 +361,8 @@ private static void createSignatureFileForRecordFile(
             // update stream digest
             LOGGER.info(MARKER, "Writing version {}", version);
             dos.writeInt(version);
-            LOGGER.info(
-                    MARKER, "Writing serializedBytes {}", hex(serializedBytes).substring(0, 32));
+            final var serializedBytesSubstring = hex(serializedBytes).substring(0, 32);
```

Comment:
Again this should only be calculated if the log level requires it (either checking it explicitly or doing this in a lambda).